src/common/commonutils/CommandUtils.c

```diff
@@ -31,7 +31,7 @@ int ExecuteCommand(void* context, const char* command, bool replaceEol, bool for
     }
     if (strlen(command) > (size_t)sysconf(_SC_ARG_MAX))
     {
-        OsConfigLogError(log, "Command too long: %s", command);
+        OsConfigLogError(log, "Command too long: %.100s...", command);
```

Comment:
> OsConfigLogError [](http://example.com/codeflow?start=7&length=17)

Do a switch and if debug logging is enabled, log the command, and if debug logging is not enabled just say command to long and number of chars?

Or, simpler, only and always say command too long, x characters long is greater than _SC_ARG_MAX (value)